src/modules/Thread/components/ThreadList/ThreadListItem.tsx

```diff
@@ -226,7 +226,7 @@ export default function ThreadListItem({
               type={LabelTypography.CAPTION_2}
               color={LabelColors.ONBACKGROUND_2}
             >
-              {format(message?.createdAt, 'MMM dd, yyyy', { locale: dateLocale })}
```

Comment:
@HoonBaek Could you please confirm if this is intended or a typo?
- channel list date separator > MMMM dd, yyyy > October 29, 2023 
- thread list date separator > MMM dd, yyyy > Oct 29, 2023

Reply:
>> Intended